### ISSUE TITLE
Training EfficientNet and EfficientNet-lite models on custom data

### ISSUE THREAD

**Issue by ameyparanjape** [tensorflow/tpu#905]
Any guidelines on how to modify hyperparameters while training the `efficientnet` and `lite` models on TPU/GPU using custom data.
What I have done:
(I am trying to follow the training tutorial on [1](https://cloud.google.com/tpu/docs/tutorials/efficientnet-2.x) and [2](https://cloud.google.com/tpu/docs/tutorials/efficientnet))
1) I have converted my custom data with 2 classes to the required tfrecord format as specified in [imagenet_to_gcs.py](https://github.com/tensorflow/tpu/blob/master/tools/datasets/imagenet_to_gcs.py)
2) Tried changing the obvious hyperparameters like `num_label_classes = 2` etc.
3) Also tried to play with the `train_batch_size` and `learning_rate`

But the model never seems to converge and loss keeps increasing. Help!!

Thank you

A sample training command looks like:
```
python3 main.py --use_tpu=False --data_dir=${DATA_DIR} --model_dir=${MODEL_DIR} --model_name='efficientnet-lite0' --mode=train --skip_host_call=true --train_steps=20000 --num_train_images=224911 --train_batch_size=1024 --iterations_per_loop=100 --num_label_classes=2
```
